mempool/src/rpc.rs

```diff
@@ -48,43 +48,61 @@ impl rpc::description::HasValueHint for GetTxResponse {
 #[rpc::describe]
 #[rpc::rpc(server, client, namespace = "mempool")]
 trait MempoolRpc {
+    /// Returns True if a transaction defined by the given id is found in the mempool.
     #[method(name = "contains_tx")]
     async fn contains_tx(&self, tx_id: Id<Transaction>) -> RpcResult<bool>;
 
+    /// Returns True if a transaction defined by the given id is found in the mempool's orphans.
+    /// An orphan transaction is a transaction with one or more inputs
     #[method(name = "contains_orphan_tx")]
     async fn contains_orphan_tx(&self, tx_id: Id<Transaction>) -> RpcResult<bool>;
 
+    /// Returns the transaction defined by the provided id, given that it is in the pool.
+    /// The returned transaction is returned in an object that contains more information about the transaction.
+    /// Returns `None` (null) if the transaction is not found.
```

Comment:
I'd probably drop the mention of `None`. This trait is only used for RPC, no need to pollute it with Rust stuff. We have separate trait for internal communication between subsystems, I contend that the Rust-related considerations should be concentrated there to the degree possible. (Multiple occurrences of this.)

Reply:
This is actually not true. We do have an internal jsonrpsee client that uses opaque communication with the node. That's how the wallet works. In that, strongly typed things are preserved.

I kinda agree with you... but I'm torn on this... because the strongly typed interface still uses rust types.